campaignion_action/src/ActionBase.php

```diff
@@ -121,11 +121,11 @@ public function redirect(Submission $submission, $delta) {
       case 'redirect':
         $redirects = Redirect::byNid($this->node->nid, $delta);
         foreach ($redirects as $r) {
-        if ($r->checkFilters($submission)) {
-          return $r->normalized();
+          if ($r->checkFilters($submission)) {
+            return $r->normalized();
+          }
         }
         break;
```

Comment:
dangling `break`. can be dropped

Reply:
I’d vote to keep it for clarity. Think of someone editing the code in the future in a way that doesn’t lead to a return in any case …

Reply:
ok with me...